.github/workflows/verify-examples.yml

```
@@ -7,6 +7,9 @@ concurrency:
   group: build-${{ github.event.pull_request.number || github.ref }}-${{github.workflow}}
   cancel-in-progress: true
 
+permissions:
+  contents: read
```

The following will make all scopes `read`, vs just contents.
```suggestion
permissions: read-all
```